src/tests/all.rs

```diff
@@ -138,11 +138,17 @@ fn app() -> (record::Bomb, Arc<App>, conduit_middleware::MiddlewareBuilder) {
     return (bomb, app, middleware);
 }
 
+/// Returnx the environment variable only if it has been defined
```

Comment:
AFAIK tests don't show up in documentation at all. While you're changing this to be a regular comment, could you remove the stray "x" at the end of "Returnx" too please? :)